src/demo-app/examples/card/fancy/card-fancy-example.html

```diff
@@ -0,0 +1,19 @@
+<md-card>
+  <md-card-header>
+    <div md-card-avatar class="header-image"></div>
+    <md-card-title>Shiba Inu</md-card-title>
+    <md-card-subtitle>Dog Breed</md-card-subtitle>
+  </md-card-header>
+  <img md-card-image src="https://upload.wikimedia.org/wikipedia/commons/7/74/Shiba_Inu_-lying.jpg">
```

Comment:
I downloaded the images so we can host them ourselves (though I don't know how to hook that up in the demo app, just including them so we can transfer them to  the docs site). Both images are public domain so using them should be fine.

https://commons.wikimedia.org/wiki/File:Shiba_inu_taiki.jpg
https://commons.wikimedia.org/wiki/File:Shiba_Inu.jpg